javascript/home.js

```diff
@@ -3,56 +3,74 @@ var portFlag = false;
 var contactFlag = false;
 var aboutFlag = false;
 $(document).ready(function(){
-  $('#portfolio').click(function(){
+  $('#portfolio > h1').click(function(){
```

Comment:
I should have caught this earlier. But the .click() is not best practice and you should use .on() I would refactor this.